### ISSUE TITLE
[Federation] Handle loadService error

### ISSUE THREAD

**Issue by oxyii** [apollographql/apollo-server#3305]
I think will be useful to handle this error in app:
https://github.com/apollographql/apollo-server/blob/0983bfabdb83ddc9c9cba00bfa447cfecd97f0ec/packages/apollo-gateway/src/loadServicesFromRemoteEndpoint.ts#L70-L75

**Comment by SethDavenport** [comment]
I agree - right now my server fails to startup if one of the downstream services is unreachable. This makes it a single-point-of-failure in our system. How cab we handle these cases with more resilience?
